modules/custom/az_eds/az_eds_user/az_eds_user.info.yml

```
@@ -0,0 +1,13 @@
+name: 'Quickstart EDS Users'
+type: module
+description: 'Provisions Drupal accounts from an LDAP source'
+core_version_requirement: ^10 || ^11 || ^12
+package: 'The University of Arizona - Experimental'
+lifecycle: experimental
+lifecycle_link: 'https://github.com/az-digital/az_quickstart/blob/main/RELEASES.md#experimental-features'
+dependencies:
+  - az_cas
+  - az_eds
```

Another suggestion in anticipation of #4990 getting merged.
```suggestion
  - az_cas:az_cas
  - az_eds:az_eds
```